disassemble/format.c

```diff
@@ -445,7 +446,7 @@ uint32_t get_indexed_element(const InstructionOperand *operand, char *outBuffer,
 	// make the "{, #<imm>}"
 	char optional_comma_and[32];
 	if(operand->immediate)
-		if(snprintf(optional_comma_and, 32, ", #%llu", operand->immediate) >= 32)
+		if(snprintf(optional_comma_and, 32, ", #%"PRIu64, operand->immediate) >= 32)
```

Comment:
is this missing a `"` ?

Reply:
No, it's doing string concatenation and we don't need to follow it with anything else.